docs/package.json

```diff
@@ -5,7 +5,7 @@
   "license": "MIT",
   "scripts": {
     "dev": "babel-node start.js",
-    "build:dev": "babel-node build.js && cd .. && yarn storybook:build && yarn examples:build",
+    "build:dev": "babel-node build.js && (cd .. && yarn storybook:build && yarn examples:build) && cp _redirects ./dist/",
```

Comment:
I think we can avoid this by putting it in `assets` which already is copied?

Reply:
The `_redirects` file need to be at the root (it's not picked up by Netlify if it's in `assets/`).